data/base-items.yaml

```diff
@@ -274,3 +274,177 @@ lumber_types:
 - tamarak # alterations
 - tamboti # ???
 - yew # longbows
+
```

Comment:
What do you guys think about a base-sorter file?

Reply:
probably better way to go about it in the end

Reply:
Yes